compiler_gym/CMakeLists.txt

```diff
@@ -49,13 +64,15 @@ cg_py_library(
   PUBLIC
 )
 
+if(COMPILER_GYM_ENABLE_LLVM_ENV)
```

Comment:
I think actually `compiler_gym::envs::llvm::service::service` is a false dependency. I don't see any reference to it in `random_search.py`. You may be able to remove this data dependency in the CMakeLists.txt/BUILD file.

Reply:
Done.